__tests__/app.test.js

```diff
@@ -33,28 +60,42 @@ describe('GET /api/topics', () => {
     })
 })
 
-describe('GET /api/not-a-route', () => {
-    it('status 404 if the endpoint is not valid', () => {
+describe('/api/articles/:article_id', () => {
+    it('GET - status 200 - respond with the correct data by article_id', () => {
         return request(app)
-            .get('/api/not-a-route')
+            .get('/api/articles/2')
+            .expect(200)
+            .then(({body}) => {
+                const {article} = body;
+
+                expect(article).toHaveProperty('author', expect.any(String));
+                expect(article).toHaveProperty('title', expect.any(String));
+                expect(article).toHaveProperty('article_id', expect.any(Number));
+                expect(article).toHaveProperty('body', expect.any(String));
+                expect(article).toHaveProperty('votes', expect.any(Number));
+                expect(article).toHaveProperty('article_img_url', expect.any(String));
+                expect(article.article_id).toBe(2);
+            })
```

Comment:
I think this should have extra properties of 'topic' and 'created_at' ?
Could we also just refactor this test please, to use a more flexible matcher such as expect.objectContaining or .toMatchObject? This will come in helpful later in the project when we add an extension to this endpoint 🙂

Reply:
good idea! understood. will try to refactor it accordingly :D